libsolidity/inlineasm/AsmParser.cpp

```diff
@@ -209,7 +214,11 @@ assembly::Statement Parser::parseElementaryOperation(bool _onlySinglePusher)
 		break;
 	}
 	default:
-		fatalParserError("Expected elementary inline assembly operation.");
+		fatalParserError(
+			m_julia ?
+			"Expected literal or identifier." :
+			"Expected elementary inline assembly operation."
```

Comment:
Can you make the error messages a bit more uniform, at least per component? Above, you added an error message that ends with `expected`, here the message starts with `expected`.